### PR TITLE
add blog post announcing Eugene as the new community rep

### PR DESCRIPTION
**Pull request by ckipp01**

### CODE REVIEW

File: blog/_posts/2023-01-10-new-ab-community-rep.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+- believe in the values and mission of the Scala Center?
+- have a broad interest in the Scala community as a whole?
+
+We considered these questions in light of what the expectation for a community
+representative are: Spokesperson for the Scala community at Advisory Board
+quarterly meetings Gathering and expressing community’s opinion on certain
+topics related to Scala Center proposals Point of contact for community
+members/groups to review and help shaping community proposals Presents and
+defends community proposals at AB meetings With all this in mind, we’re happy to
+introduce Eugune, who has been a pillar in the Scala community for years. Mostly
```
Comment (Philippus) [inline blog/_posts/2023-01-10-new-ab-community-rep.md:41]:
```suggestion
introduce Eugene, who has been a pillar in the Scala community for years. Mostly
```

File: blog/_posts/2023-01-10-new-ab-community-rep.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+known for:
+
+- Being the lead maintainer of [sbt][sbt]
+- His [blog][eugene-blog], writing, and [tutorials][herding-cats]
+- His many talks and workshops given at various conferences
+- Community work in helping organize the [Scala Matsuri][scalamatsuri] conferences
+- His endless mentoring online and beyond
+- His tireless involvement and focus on ensuring diversity and inclusion are at the forefront of events and in our community, ultimately making Scala a safer more welcoming environment for all
+
+We trust that Eugune will be a valuable addition to the board both for his
```
Comment (Philippus) [inline blog/_posts/2023-01-10-new-ab-community-rep.md:51]: both is for two things. :)

```suggestion
We trust that Eugune will be a valuable addition to the board for his
```

**Comment by SethTisue** [comment]
should the post say something about the _other_ empty community repo post? I feel like it's a bit weird to say nothing at all, not even a "we're still working on filling the other seat"

**Comment by darjutak** [comment]
> should the post say something about the _other_ empty community repo post? I feel like it's a bit weird to say nothing at all, not even a "we're still working on filling the other seat"

As per Scala Center bylaws, there's no obligation to have more than 1 representative. For now, we start with one and when we have more, we will announce that.

File: blog/_posts/2023-01-10-new-ab-community-rep.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+We considered these questions in light of what the expectation for a community
+representative are:
+
+- Spokesperson for the Scala community at Advisory Board quarterly meetings
+- Gathering and expressing community’s opinion on certain topics related to Scala Center proposals
+- Point of contact for community members/groups to review and help shaping community proposals
+- Presents and defends community proposals at AB meetings
+
+With all this in mind last November we made the decision on the nomination, and
+we’re happy to introduce Eugene, who has been a pillar in the Scala community
```
Comment (eed3si9n) [inline blog/_posts/2023-01-10-new-ab-community-rep.md:44]: Thanks for this introduction. A pillar sounds too much, maybe "an unusual enthusiast"?

Comment (ckipp01) [reply blog/_posts/2023-01-10-new-ab-community-rep.md:44]: I have no problem with the word pillar, but you're 100% free to veto me 😆 ... seeing that this is you we're talking about. How about `a dedicated enthusiast`? Would that be ok with you?

Comment (eed3si9n) [reply blog/_posts/2023-01-10-new-ab-community-rep.md:44]: Yea. I think that's better. Thanks!

**Comment by ckipp01** [comment]
Verified with Darja that this is ok to merge, so I'm going to go ahead.

**Comment by SethTisue** [comment]
@softinio points out that we forgot to mention Eugene's cooking prowess :-)

**Pull request merged**